src/scenes/Fog.js

```diff
@@ -1,27 +1,27 @@
 import { Color } from '../math/Color.js';
 
-function Fog( color, near, far ) {
+class Fog {
 
-	this.name = '';
+	constructor( color, near, far ) {
 
-	this.color = new Color( color );
+		this.name = '';
 
-	this.near = ( near !== undefined ) ? near : 1;
-	this.far = ( far !== undefined ) ? far : 1000;
+		this.isFog = true;
```

Comment:
As per https://github.com/mrdoob/three.js/pull/19997#issuecomment-674021475, we should do this instead:

```js
Fog.prototype.isFog = true;
```

Reply:
I'll need to go back over my PR's and adjust accordingly